firestore-bigquery-export/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## Version 0.1.2
+
+fixed - Added "IF NOT EXISTS" to safely run script multiple times
+fixed - Updated BigQuery library
```

Comment:
Can you please add a reference to #197 ?

Suggested change to the message:

```
Updated BigQuery dependency in fs-bq-import-collection package.json (#197)
```

cc @IanWyszynski

Reply:
Is this suggestion for the first or the second bullet?

Reply:
the second bullet. first bullet should reference #193

Reply:
I agree with @jhuleatt here.

Reply:
updated directly in file :-)